core/build.gradle

```diff
@@ -133,10 +133,10 @@ test {
 }
 
 def genExamples = tasks.register("genExamples", Exec) {
-    inputs.files(fileTree('examples/generated/')).withPropertyName('sourceFiles')
+    workingDir '../python/railjson_generator/railjson_generator/'
+    inputs.files(fileTree('../python/railjson_generator/')).withPropertyName('sourceFiles')
```

Comment:
It feels weird to have relative paths here. Maybe we could add a way to generate infras without specifying the full path of the input scripts?